docs/Learning-Environment-Heuristic-Brains.md

```diff
@@ -0,0 +1,22 @@
+# Heuristic Brain
```

Comment:
Shouldn't this file be called `Learning-Environment-Design-Heuristic-Brains.md` ?